packages/ui/components/DropdownMenu.tsx

```diff
@@ -18,6 +18,9 @@ interface DropdownMenuProps {
   items: DropdownMenuItem[]
   side?: 'top' | 'right' | 'bottom' | 'left'
   align?: 'start' | 'center' | 'end'
+  multiple?: boolean
+  selectedOptions?: string[]
```

Comment:
what is the point of adding these here if we are not using them?

Reply:
In the end, we surely do. but for now I think it makes sense to remove these from the interface since the entire `DropDownMenu` is not handling these options.

Reply:
@Gmin2 can you remove the changes to this component?

Reply:
Done @KhudaDad414